pkg/envoy/builders/accesslog/accesslog.go

```diff
@@ -0,0 +1,72 @@
+package accesslog
+
+import (
```

Comment:
Should this all code be in `xds/listeners/v3` where we have all configurers?

Reply:
No :) I'm planning to add README.md for the `pkg/envoy/builders` when I'm done with the MeshAccessLog. But the idea is to have these new Builders and Configurers as a replacement. They're simpler and way less verbose than previous implementation in `xds/listeners` (mostly thanks to generics). 

Also, I'll mention in the README.md: 
* we need to follow the directory structure as in https://github.com/envoyproxy/envoy/tree/main/api/envoy/config
* we shouldn't depend on any dirs from `pkg/plugins/policies`